main/realmain.cc

```diff
@@ -769,6 +766,14 @@ int realmain(int argc, char *argv[]) {
             // Second run: all the other files (the packageDB shouldn't change)
             pipeline::package(*gs, absl::Span<ast::ParsedFile>(nonPackageIndexed), opts, *workers);
 
+            {
+                // TODO(jez) Put this back after the call to `pipeline::package` in the stripePackages section
+                // Only need to compute hashes when running to compute a FileHash
+                auto foundHashes = nullptr;
+                auto canceled = pipeline::name(*gs, absl::Span<ast::ParsedFile>(indexed), opts, *workers, foundHashes);
```

Comment:
Is this really correct?  We're going to name all the indexed files, and then, shortly thereafter, we're going to name the non-packaged index files?  Would it be any more efficient to unpartition them, and name everything all at once?

Reply:
Technically yes, but I'm trying to preserve as much of the old structure of the code as possible because it was intentional to separate things the way they were.